public/pages/HistoricalDetectorDetail/components/HistoricalDetectorConfig/HistoricalDetectorConfig.tsx

```diff
@@ -84,9 +84,13 @@ export const HistoricalDetectorConfig = (
         <EuiFlexItem>
           {renderCell(
             'Date range',
-            renderDate(get(props, 'detector.detectionDateRange.startTime', 0)) +
+            renderDate(
+              get(props, 'detector.detectionDateRange.startTime', undefined)
```

Comment:
minor: I think `get` will return undefined by default if `detector.detectionDateRange.startTime` is missing.